### DISCUSSION TITLE
Fatal: cannot lock ref 'refs/heads/test'

### DISCUSSION THREAD

**Discussion by Lucas-Erkana** [community/community#67536] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Trying to create a new branch and it keeps showing this error:
![image](https://github.com/community/community/assets/41428579/fd629cf5-0bf3-4de8-80b7-7cf695a09dc3)
fatal: cannot lock ref 'refs/heads/test': Unable to create 'C:/Users/Lu-cash/Documents/GithubChanllenges/Algorithms/.git/refs/heads/test.lock': No such file or directory

I removed it and tried to clone it and then it showed this error:
![image](https://github.com/community/community/assets/41428579/08a2b4f8-3ff1-4c04-ac57-424aa922eb5d)
fatal: could not create work tree dir 'Algorithms': No such file or directory

[Repo](https://github.com/Lucas-Erkana/Algorithms)

Has anyone ever encountered this error?

**Comment by zhorabay** [comment]
When I pulled your repo and tried to create a branch it worked. 
You can read this, if it helps: https://stackoverflow.com/questions/45141188/git-fatal-cannot-lock-ref
